### DISCUSSION TITLE
Amiga version contains Mac references

### DISCUSSION THREAD

**Discussion by djipi** [Croquetx/thecolony#5] [General]
There are various references to the Mac in the Amiga source code.
By example in the Colony-Amiga/source/inits.c source, in inits function, there is a memory detection to set a Mac512 or Mac2 variable.

**Comment by Croquetx** [comment]
The Amiga was a port from the Mac version. I don't remember if the Amiga had different memory setups, but it was probably doing the same thing there. This port was done 24 years ago.
